test/parallel/test-path-resolve.js

```diff
@@ -69,3 +69,11 @@ if (common.isWindows) {
   const resolvedPath = spawnResult.stdout.toString().trim();
   assert.strictEqual(resolvedPath.toLowerCase(), process.cwd().toLowerCase());
 }
+
+if (!common.isWindows) {
+  // Test handling relative paths to be safe when process.cwd() fails.
+  process.cwd = () => '';
```

Comment:
This may seem like an unnecessary request, and feel free to ignore me, but maybe could you add `assert.strictEqual(process.cwd(), '')` below this line, to make sure that overriding actually worked? I could see a Node.js collaborator coming along and making `process.cwd()` non-writable, in which case this assignment would fail silently…

Reply:
Done, thanks, I didn't think about it

Reply:
I'm going to squash this into the original commit ;-)